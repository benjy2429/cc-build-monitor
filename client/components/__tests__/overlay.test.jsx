import React from 'react';
import { shallow } from 'enzyme';
import Overlay from '../overlay';

const defaultProps = {
  title: 'The title',
  subtitle: 'The subtitle',
};

describe('Overlay', () => {
  it('renders correctly', () => {
    const component = shallow(
      <Overlay {...defaultProps} />,
    );
    expect(component).toMatchSnapshot();
  });
});
