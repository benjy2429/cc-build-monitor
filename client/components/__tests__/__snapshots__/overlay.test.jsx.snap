// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay renders correctly 1`] = `
<div
  className="overlay"
>
  <div
    className="overlay-wrapper"
  >
    <div
      className="overlay-title"
    >
      The title
    </div>
    <div
      className="overlay-subtitle"
    >
      The subtitle
    </div>
  </div>
</div>
`;
